packages/contracts/contracts/test/DAOMock.sol

```diff
@@ -69,4 +69,10 @@ contract DAOMock is IDAO, PermissionManager {
     ) external pure override returns (bytes4) {
         return 0x0;
     }
+
+    function registerStandardCallback(
```

Comment:
What's the main goal behind this new function on the DAO contract?

Reply:
This dynamically can register callbacks (such as `onERC721Received` which can be found on ERC721 contract) so dao can receive erc721 e.x